xr_examples/hello_xr.py

```diff
@@ -1,4 +0,0 @@
-from xr_examples.hello_xr.main import main
```

Comment:
I still want to use this file during development and testing; even if it's not intended for external users.